.eslintrc.js

```diff
@@ -55,7 +55,6 @@ module.exports = {
     '@typescript-eslint/member-delimiter-style': 'off',
     '@typescript-eslint/type-annotation-spacing': 'off',
     '@emotion/jsx-import': 'error',
-    '@emotion/no-vanilla': 'error',
```

Comment:
This is a change that will be reflected in the latest version of canvas-kit, but it's not in this branch yet (and I needed it).